src/ini_load.cpp

```diff
@@ -29,7 +29,7 @@ IniItem::IniItem(const std::string &name)
  * Replace the current value with another value.
  * @param value the value to replace with.
  */
-void IniItem::SetValue(const std::string_view value)
+void IniItem::SetValue(std::string_view value)
```

Comment:
Why remove the `const`?

Reply:
std::string_view is read only, and passed by value, so const or not does not make a difference.

So I made it consistent.